src/middlewared/middlewared/alembic/versions/12.0/2020-05-18_14-29_nmbclusters-fix.py

```diff
@@ -0,0 +1,98 @@
+"""remove nmbclusters sysctl
+
+Revision ID: 730c995cbd37
+Revises: 434ea5397cd3
+Create Date: 2020-05-18 14:29:59.887895+00:00
+
+"""
+from alembic import op
+import subprocess as subp
+from middlewared.utils import osc
+
+
+# revision identifiers, used by Alembic.
+revision = '730c995cbd37'
+down_revision = '434ea5397cd3'
+branch_labels = None
+depends_on = None
+
+
+def get_db_sysctl_value():
+    TABLE = 'system_tunable'
+    SELECT_FROM = f'SELECT * FROM {TABLE}'
+    WHERE = ' WHERE tun_var = "kern.ipc.nmbclusters"'
+
+    conn = op.get_bind()
+
+    db_data = conn.execute(f'{SELECT_FROM} {WHERE}').fetchall()
+
+    data = {}
+    for i in db_data:
+        # we only care if this is a sysctl type value
+        # because a loader type will not have been
+        # set by the autotune script and would have
+        # been explicitly set by the end-user so leave
+        # it alone otherwise
+        if i['tun_type'] == 'sysctl':
+            data['value'] = i['tun_value']
+            data['id'] = i['id']
+            return data
+
+    return data
+
+
+def get_os_sysctl_value():
+
+    curr_val = None
+
+    cp = subp.run(
+        ['sysctl', 'kern.ipc.nmbclusters'],
+        stdout=subp.PIPE,
+        stderr=subp.PIPE,
+    )
+
+    if cp.returncode:
+        return curr_val
+
+    if cp.stdout:
+        try:
+            curr_val = int(cp.stdout.decode().strip().split(':')[-1])
+        except Exception:
+            pass
+
+    return curr_val
+
+
+def remove_db_entry(db_value, os_value):
+    TABLE = 'system_tunable'
+    DELETE_FROM = f'DELETE FROM {TABLE}'
+    WHERE = f' WHERE id = {db_value["id"]}'
```

Comment:
We shouldnt really miss uppercase which usually means global vars within a method scope. These vars also seems superfluous and do not bring any value

Perhaps TABLE could be global and the rest just part of execute string?

Reply:
yep, agreed. I fixed.